runtime/oti/j9port.h

```diff
@@ -779,6 +779,8 @@ typedef struct J9CacheInfoQuery {
 #define J9PORT_CTLDATA_MEM_CATEGORIES_SET OMRPORT_CTLDATA_MEM_CATEGORIES_SET
 #define J9PORT_CTLDATA_AIX_PROC_ATTR OMRPORT_CTLDATA_AIX_PROC_ATTR
 #define J9PORT_CTLDATA_ALLOCATE32_COMMIT_SIZE OMRPORT_CTLDATA_ALLOCATE32_COMMIT_SIZE
+#define J9PORT_CTLDATA_ALLOCATE32_INCREMENT_SIZE OMRPORT_CTLDATA_ALLOCATE32_INCREMENT_SIZE
+#define J9PORT_CTLDATA_ALLOCATE32_QUICK_ALLOC OMRPORT_CTLDATA_ALLOCATE32_QUICK_ALLOC
```

Comment:
I don't think we should be adding to this list of macros: just use the ones provided by OMR.

Reply:
The macro defined on the line above, i.e.:
```
#define J9PORT_CTLDATA_ALLOCATE32_INCREMENT_SIZE OMRPORT_CTLDATA_ALLOCATE32_INCREMENT_SIZE
```
is also not used. Should that be removed too?